pages/cart/step1.default.js

```diff
@@ -1,4 +1,4 @@
-import { useEffect, useState } from "react";
+import {useEffect, useRef, useState} from "react";
```

Comment:
este daj na zaciatok aj to 'use client' stare fronty by to mali ignorovat a novy bude spravne fungovat. Ale OTESTUJ to, ci stary nepada.

Reply:
@ivankopcik zapracované